scico/flax/inverse.py

```diff
@@ -463,3 +464,53 @@ def __call__(self, y: Array, train: bool = True) -> Array:
             x = block(alpha_ini=alpha0_i)(x, y, train)
             alpha0_i /= 2.0
         return x
+
+
+@partial(jit, static_argnums=0)
+def power_iteration(A: LinearOperator, maxiter: int = 100):
+    """Compute largest eigenvalue of a diagonalizable :class:`.LinearOperator`.
+
+    Compute largest eigenvalue of a diagonalizable
+    :class:`LinearOperator` using power iteration.
+    The function makes use of lax functionality to
+    allow jitting and general jax function composition.
+
```

Comment:
A part of the story is missing: This function has the same functionality as :class:\`.linop.power_iteration\`, but is implemented using lax operations to allow jitting ...

Also, the docstring formatting doesn't look quite right (lines are too short).

Reply:
Added the suggested information and re-formatted the corresponding docstring.